File: Python/Ex_35.py
values = []

for c in range(0, 20):
    v = float(input("Enter with a value: "))
    values.append(v)

multiplier = float(input('Enter with a multiplier: '))

for i in range(0, 20):
    values[i] *= multiplier

for value in values:
    print(value)
 

File: c/Ex_35.c
#include <stdio.h>

int main(int argc, char const *argv[])
{
    int values[20];
    float multiplier;

    for (int i = 0; i < 20; i++)
    {
        printf("Enter with a value: ");
        scanf("%i", values[i]);
    }

    printf("Enter with a multiplier: ");
    scanf("%f", &multiplier);

    for (int i = 0; i < 20; i++)
    {
        values[i] *= multiplier;
    }

    for (int i = 0; i < 20; i++)
    {
        printf(values[i]);
    }
    
    return 0;
}
